sklearn/naive_bayes.py

```diff
@@ -708,6 +708,112 @@ def _joint_log_likelihood(self, X):
                 self.class_log_prior_)
 
 
+class MultinomialCNB(BaseDiscreteNB):
+    """
```

Comment:
Fixed. FYI, the [short description for `MultinomialNB` is not PEP257 compliant](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/naive_bayes.py#L611) (I was modifying a copied version of that class).